src/Symfony/Component/Security/Http/EventListener/IsCsrfTokenValidAttributeListener.php

```diff
@@ -49,4 +56,18 @@ public static function getSubscribedEvents(): array
     {
         return [KernelEvents::CONTROLLER_ARGUMENTS => ['onKernelControllerArguments', 25]];
     }
+
+    private function getIsCsrfTokenValidId(string|Expression $id, Request $request, array $arguments): string
+    {
+        if ($id instanceof Expression) {
+            $this->expressionLanguage ??= new ExpressionLanguage();
+
+            return (string) $this->expressionLanguage->evaluate($id, [
```

Comment:
What happends if the expression evaluates to "true" ?

Reply:
I'd say that the token id will end up being `1`. The goal of the expression is to build the token id with the possibility to access the controller parameters

Reply:
The token id beeing a string right ?

Reply:
yes, `"1"`